librz/core/cmd_descs/cmd_cmp.yaml

```diff
@@ -0,0 +1,145 @@
+# SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+# SPDX-License-Identifier: LGPL-3.0-only
+---
+name: cmd_cmp
+commands:
+  - name: c
+    summary: Compare an escaped <string> with data at current offset
+    cname: cmd_cmp_string
+    type: RZ_CMD_DESC_TYPE_ARGV_MODES
+    modes:
+      - RZ_OUTPUT_MODE_STANDARD
+      - RZ_OUTPUT_MODE_JSON
+    args:
+      - name: string
+        type: RZ_CMD_ARG_TYPE_STRING
+  - name: c1
+    summary: Compare 8-bit data at current offset with the data at <addr>
+    cname: cmd_cmp_bits
+    args:
+      - name: addr
+        type: RZ_CMD_ARG_TYPE_RZNUM
+  - name: ca
+    summary: Compare <n> bytes of data at <addr> with the data at current offset (also return in $?)
+    cname: cmd_cmp_addr
+    args:
+      - name: addr
+        type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: n
+        type: RZ_CMD_ARG_TYPE_NUM
+  - name: cb
+    summary: Compare <n> (upto 8) bytes at current offset with a number <num> (also return in $?)
```

Comment:
Is this a comparison of bytes or of "values"?

Reply:
It will compare the bytes at a memory location with the number `<num>`. So, if a memory location has the following bytes: 0xff32, and we input `<num>` as 0x10 and `<n>` as 2, it will show the comparison as follows (assuming little-endian):
 
 ```
0x32 -- 0x10
0xff -- 0x00
 ```